File: Client/src/app/model/ArticleModel.ts
import { InfoModel } from './InfoModel';
import { CommentModel } from './CommentModel';

export class ArticleModel {
    _id: string;
    title: string;
    descritpion: string;
    body: string;
    info: InfoModel;
    isHidden: boolean;

    comments: CommentModel[];
}

File: Server/src/Server/Model/ViewModels/ArticleModel.cs
using System.Collections.Generic;

namespace Server.ViewModels.Model
{
    public class ArticleModel
    {
        public string _id;
        public string Title;
        public string Description;
        public string Body;
        public InfoModel Info;
        public bool IsHidden;

        public List<CommentModel> Comments;
    }
}
